### DISCUSSION TITLE
Renovate not make pull requests.

### DISCUSSION THREAD

**Discussion by doncl** [renovatebot/renovate#19798] [Request Help]
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "labels": ["dependencies"],
  "prHourlyLimit": 5,
  "prConcurrentLimit": 5,
  "hostRules": [
    {
      "matchHost": "api.github.com",
      "concurrentRequestLimit": 1,
      "maxRequestsPerSecond": 8
    }
  ]
}

```

I've been making personal repos, and installing the app, and it was working....until sometime today.   I started a new repo, and no dice.   Deleted and started again (create the repo, add an iOS project, build a Podfile, commit everything); the renovate/configure branch gets created, and a PR for it, I edit renovate.json with the above config, the PR adjusts to show it's a custom config, and I merge it....and....Radio Silence.

I know this is some stupid cockpit error, but it was working for me before.

**Comment by rarkins** [comment]
Is it the hosted app? And have you checked app.renovatebot.com for logs?
